audio/audio.py

```diff
@@ -166,6 +169,23 @@ def studio_submit(self, request, suffix=''):
             storage.save(file_path, File(transcript_file.file))
             self.transcript_file = file_path
 
+        will_upload_alt_transcript_file = 'alt_transcript_file' in data and hasattr(data.get('alt_transcript_file'), 'file')
+
+        if data.get("delete_alt_transcript_file", "") == "on" or will_upload_alt_transcript_file:
+            if self.alt_transcript_file and storage.exists(self.alt_transcript_file):
+                storage.delete(self.alt_transcript_file)
+            self.alt_transcript_file = None
+
+        if will_upload_alt_transcript_file:
+            alt_transcript_file = data['alt_transcript_file']
+
+            # generate a safe path for the alt_transcript file
+            name = get_valid_filename(alt_transcript_file.filename)
+            safe_usage_key = get_valid_filename(self.usage_key)
+            file_path = f"{safe_usage_key}/alt_transcripts/{name}"
+            storage.save(file_path, File(alt_transcript_file.file))
```

Comment:
I'm saving this in django storage, in the same way as the standard vtt transcript file above. I'm not convinced this pattern is the best way for saving files for xblocks; I asked on https://discuss.openedx.org/t/recommended-method-for-uploading-files-for-xblocks/17642 .

Either way, changing it will involve some refactoring and potentially breaking changes for the vtt transcript file, so I think following this pattern as a precedent in this xblock is fine for now.